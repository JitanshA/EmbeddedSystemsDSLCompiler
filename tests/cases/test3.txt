SET_PIN(5, HIGH);
int status = READ_PIN(7);
