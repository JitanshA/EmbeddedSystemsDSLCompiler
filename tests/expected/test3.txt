GPIO_SET 5, HIGH
GPIO_READ 7, status
